src/vg_chain_state.erl

```diff
@@ -25,14 +26,24 @@
 start_link() ->
     gen_server:start_link({local, ?SERVER}, ?MODULE, [], []).
 
+next() ->
+    gen_server:call(?SERVER, next).
+
 init([]) ->
     Replicas = list_to_integer(application:get_env(vonnegut, replicas, "1")),
     {ok, #state{replicas=Replicas,
                 active=false}, 0}.
 
+handle_call(next, _From, State = #state{next_node = Next}) ->
+    {reply, Next, State};
 handle_call(_, _From, State) ->
     {noreply, State}.
 
+handle_cast({route_write, Sender, Topic, Partition, MessageSet}, State) ->
```

Comment:
this should be removed, it's from a previous version where I used this as a jump server to get to the next node in the chain.